### PR TITLE
 add get metric command

### PR DESCRIPTION
**Pull request by pkarkazis**

### CODE REVIEW

File: src/tnglib/monitor.py [hunk 1]
```diff
@@ -160,3 +160,35 @@ def get_metrics(vnf_uuid, vdu_uuid):
...
+        error = resp.text
+        return False, error
+
+    templates = json.loads(resp.text)
+
+    temp_res = []
+
+    if 'metrics' in templates and  'result' in templates['metrics']:
+        for res in templates['metrics']['result']:
+            dic = {'job': res['metric']['job'],'instance': res['metric']['instance'],'value': res['value'][1]
```
Comment (tsoenen) [inline src/tnglib/monitor.py]: Hi Panos. We are trying to be conform pep8 python standard, so line lengths can't exceed 80 chars. Can you split this one up please?

File: src/tnglib/monitor.py [hunk 2]
```diff
@@ -160,3 +160,35 @@ def get_metrics(vnf_uuid, vdu_uuid):
         LOG.debug("Request returned with " + (json.dumps(templates)))
         error = "VDUs not found"
         return False, error
+
+def get_metric(metric_name):
+    """
+    Returns value per metric name.
+
+    """
+    resp = requests.get(env.monitor_api+'/prometheus/metrics/name/'+metric_name,
```
Comment (tsoenen) [inline src/tnglib/monitor.py]: Hi Panos. We are trying to be conform pep8 python standard, so line lengths can't exceed 80 chars. Can you split this one up please?

File: src/tngcli/cli.py [hunk 3]
```diff
@@ -188,7 +188,7 @@ def dispatch(args):
     
     #monitor subcommand
     elif args.subparser_name == 'monitor':
-        sel_args = [args.target_list, args.service_list, args.metric_list, args.vnf_uuid, args.vdu_uuid]
+        sel_args = [args.target_list, args.service_list, args.metric_list, args.vnf_uuid, args.vdu_uuid, args.metric_name]
```
Comment (tsoenen) [inline src/tngcli/cli.py]: Hi Panos. We are trying to be conform pep8 python standard, so line lengths can't exceed 80 chars. Can you split this one up please?

Comment (pkarkazis) [reply src/tngcli/cli.py]: Hi Thomas, I think that its ok now

**Pull request merged**